R/calcualte_age.R

```diff
@@ -21,78 +26,88 @@
 #'
 #' @examples
 #' age <- calculate_age(
-#'   data          = readRDS(system.file("extdata", "test_df.RDS",
-#'                                       package = "cleanepi")),
+#'   data = readRDS(system.file("extdata", "test_df.RDS",
+#'     package = "cleanepi"
+#'   )),
 #'   target_column = "dateOfBirth",
-#'   end_date      = Sys.Date(),
-#'   age_in        = "months",
-#'   na_strings    = "-99"
+#'   end_date = Sys.Date(),
+#'   age_in = "months",
+#'   age_remainder_unit = "days",
+#'   na_strings = "-99"
 #' )
 calculate_age <- function(data,
                           target_column = NULL,
-                          end_date      = Sys.Date(),
-                          age_in        = "years",
-                          na_strings    = cleanepi::common_na_strings) {
+                          end_date = Sys.Date(),
+                          age_in = "years",
+                          na_strings = common_na_strings,
+                          age_column_name = sprintf("age_%s", age_in),
+                          age_remainder_unit = c("days", "weeks", "months")) {
```

Comment:
I think the reverse order is more natural.i.e., months -> weeks -> days.

Reply:
Agreed - will change this. But see also my general question of why the remainder needs to be provided separately, and why the age should not be calculated as a numeric value directly?

Reply:
Change in https://github.com/epiverse-trace/cleanepi/pull/83/commits/0c647d294f55ee3e17cb679950cdd28390da82f5

Reply:
Thanks @jamesmbaazam - I have added some tests in a previous commit that @pratikunterwegs mentionned.

@pratikunterwegs - about the remainder been sent in months, weeks or days: this was a suggestion from some people at the MoH during the showcase we organised with them. Having this feature and the `digits` feature will capture all the evantual needs from users about age calculation.